### ISSUE TITLE
Is it possible to add parking spaces and crosswalk？

### ISSUE THREAD

**Issue by runningkoala** [daohu527/imap#71]

**Comment by daohu527** [comment]
No, this tool just does a conversion, if opendrive have this info then we can do it. But imap does not support these functions yet, If you have the corresponding map send me, we can support

**Comment by runningkoala** [comment]
ok, please give me your email address.

**Comment by daohu527** [comment]
The email daohu527@gmail.com, and we'd better make sure the map complies with opendrive specification
